### DISCUSSION TITLE
Can we have more poses?

### DISCUSSION THREAD

**Discussion by rebeccakzpt** [naverz/zepeto-studio-kor#752] [4. General]
Hi there! I'm making a world for people to enjoy themselves and take pictures everywhere (setting poses for players), but the given poses are so few. Where should we get some more existing Zepeto poses? Do we have to make them manually?

**Comment by smilejsu82** [comment]
> Hi there! I'm making a world for people to enjoy themselves and take pictures everywhere (setting poses for players), but the given poses are so few. Where should we get some more existing Zepeto poses? Do we have to make them manually?

You can apply custom animation using mixamo
https://www.mixamo.com/#/

#538

**Reply by rebeccakzpt** [reply]
Thank you! But they are all animations... Can we get static postures too?
